packages/flutter/lib/src/material/typography.dart

```diff
@@ -256,7 +320,7 @@ class Typography extends Diagnosticable {
   @override
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
-    final Typography defaultTypography = Typography();
+    final Typography defaultTypography = Typography.material2018();
```

Comment:
Does making the default 2018 here change anything?

Reply:
Good catch! Currently the default value for is the 2014 text theme.